src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -60,6 +60,131 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
+  Gets the number of times there was contention when trying to take the monitor's
+  lock.
+* [ThreadPool.ThreadCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.threadcount?view=netcore-3.1):
+  Gets the number of thread pool threads that currently exist.
+* [ThreadPool.CompletedWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.completedworkitemcount?view=netcore-3.1):
+  Gets the number of work items that have been processed so far.
+* [ThreadPool.PendingWorkItemCount](https://docs.microsoft.com/dotnet/api/system.threading.threadpool.pendingworkitemcount?view=netcore-3.1):
+  Gets the number of work items that are currently queued to be processed.
+* [Timer.ActiveCount](https://docs.microsoft.com/dotnet/api/system.threading.timer.activecount?view=netcore-3.1):
+  Gets the number of timers that are currently active. An active timer is registered
+  to tick at some point in the future, and has not yet been canceled.
+
+### Assemblies related metrics
+
+| Name                                      | Description                                              | Units          | Instrument Type | Value Type | Attribute Key(s) | Attribute Values |
+|-------------------------------------------|----------------------------------------------------------|----------------|-----------------|------------|------------------|------------------|
+| process.runtime.dotnet.**assembly.count** | The number of .NET assemblies that are currently loaded. | `{assemblies}` | ObservableGauge | `Int64`    |                  |                  |
+
+The API called in this section is:
+
+* [AppDomain.GetAssemblies](https://docs.microsoft.com/dotnet/api/system.appdomain.getassemblies):
+  Gets the number of the assemblies that have been loaded into the execution context
+  of this application domain.
+
+### Exception counter metric
+
+| Name                                       | Description                                                                                                                                                                                                            | Units      | Instrument Type   | Value Type | Attribute Key(s) | Attribute Values |
+|--------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------|-------------------|------------|------------------|------------------|
+| process.runtime.dotnet.**exception.count** | Count of exceptions that have been thrown in managed code, since the observation started. The value will be unavailable until an exception has been thrown after OpenTelemetry.Instrumentation.Runtime initialization. | `{timers}` | ObservableCounter | `Int64`    |                  |                  |
+
+The API called in this section is:
```

Comment:
Since there is no API to get exception.count, i think this section is misleading. We are using this, and doing the count using Metrics Counter API here

Reply:
I've added a note here as explanation.